src/test/java/fi/otavanopisto/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -0,0 +1,243 @@
+      download(getFullUrl(String.format("/CodeList/GetAreaCodes/type/%s", areaCodeType)), new File(getCodesFolder(), String.format("%s.json", areaCodeType.toLowerCase())));
+    }
+
+    download(getFullUrl("/CodeList/GetCountryCodes"), new File(getCodesFolder(), "country.json"));
+    download(getFullUrl("/CodeList/GetLanguageCodes"), new File(getCodesFolder(), "language.json"));
+    download(getFullUrl("/CodeList/GetMunicipalityCodes"), new File(getCodesFolder(), "municipality.json"));
+    download(getFullUrl("/CodeList/GetPostalCodes"), new File(getCodesFolder(), "postal.json"));
+
+    return this;
+  }
+  
+  @SuppressWarnings("unused")
+  private void printIds() throws IOException {
+    File[] organizationFiles = listJsonFiles(getOrganizationsFolder());
+    File[] serviceFiles = listJsonFiles(getServicesFolder());
+    File[] serviceChannelFiles = listJsonFiles(getServiceChannelsFolder());
+    
+    printIdList("ORGANIZATIONS", organizationFiles);
+    printIdList("SERVICES", serviceFiles);
+    printIdList("SERVICE_CHANNELS", serviceChannelFiles);
+    
+    for (ServiceChannelType type : ServiceChannelType.values()) {
+      printServiceChannelList(type, serviceChannelFiles);
+    }
+  }
+  
+  private ServiceChannelType resolveChannelType(String type) {
+    switch (type) {
+      case "EChannel":
+        return ServiceChannelType.ELECTRONIC_CHANNEL;
+      case "ServiceLocation":
+        return ServiceChannelType.SERVICE_LOCATION;
+      case "PrintableForm":
+        return ServiceChannelType.PRINTABLE_FORM;
+      case "Phone":
+        return ServiceChannelType.PHONE;
+      case "WebPage":
+        return ServiceChannelType.WEB_PAGE;
+      default:
+        return null;
+    }
+  }
+
+  private void printIdList(String constName, File[] files) {
+    List<String> ids = new ArrayList<>();
+    for (File file : files) {
+      ids.add(String.format("      \"%s\"",  StringUtils.stripEnd(file.getName(), JSON)));
+    }
+    
+    System.out.println(String.format("\n    public final static String[] %s = {", constName));
+    System.out.println(StringUtils.join(ids, ",\n"));
+    System.out.println("    };");
+  }
+
+  private void printServiceChannelList(ServiceChannelType type, File[] files) throws IOException {
+    List<String> ids = new ArrayList<>();
+    for (File file : files) {
+      ServiceChannelTypeExtract serviceChannelTypeExtract = getObjectMapper().readValue(file, ServiceChannelTypeExtract.class);
+      if (resolveChannelType(serviceChannelTypeExtract.getServiceChannelType()) == type) {
+        ids.add(String.format("      \"%s\"",  StringUtils.stripEnd(file.getName(), JSON)));
+      }
+    }
+    
+    System.out.println(String.format("\n    public final static String[] %s_SERVICE_CHANNELS = {", type.name()));
+    System.out.println(StringUtils.join(ids, ",\n"));
+    System.out.println("    };");
+  }
+
+  private PtvDownload downloadOrganizations() throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "downloadOrganizations" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)